### PR TITLE
[DT-506][risk=no] cleanup environment variables for tanagra deployment

### PR DESCRIPTION
**Pull request by freemabd**
Cleanup environment variables for tanagra deployment

- removed duplicate database url/username/passwords from vars.env
- moved non sensitive variables into environments.rb

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

**Review by dolbeew — APPROVED** [review]

File: tanagra-aou-utils/tanagra_env_variables_template.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  TANAGRA_DB_SOCKET_FACTORY: com.google.cloud.sql.mysql.SocketFactory
+  TANAGRA_DB_URI: jdbc:mysql:///tanagra_db
+  TANAGRA_DB_USERNAME: $WORKBENCH_DB_USER
+  TANAGRA_ACCESS_CONTROL_BASE_PATH: https://$TANAGRA_ACCESS_CONTROL_BASE_PATH
+  TANAGRA_ACCESS_CONTROL_MODEL: $TANAGRA_ACCESS_CONTROL_MODEL
+  TANAGRA_AUTH_GCP_PROJECT_ID: $TANAGRA_AUTH_GCP_PROJECT_ID
+  TANAGRA_UNDERLAY_FILES: $TANAGRA_UNDERLAY_FILES
+  TANAGRA_AUTH_DISABLE_CHECKS: $TANAGRA_AUTH_DISABLE_CHECKS
+  TANAGRA_AUTH_BEARER_TOKEN: $TANAGRA_AUTH_BEARER_TOKEN
+  TANAGRA_AUTH_IAP_GKE_JWT: false
```
Comment (chenchals) [inline tanagra-aou-utils/tanagra_env_variables_template.yaml:14]: why do I need this?
TANAGRA_AUTH_IAP_GKE_JWT -> is this setting those 3-auth types to false?
TANAGRA_AUTH_GCP_PROJECT_ID

File: api/libproject/environments.rb [hunk 2]
```diff
@@ -47,6 +47,10 @@ def env_with_defaults(env, config)
     :publisher_account => "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com",
     :source_cdr_project => "all-of-us-ehr-dev",
     :source_cdr_wgs_project => "all-of-us-workbench-test",
+    :tanagra_access_control_model => "OPEN_ACCESS",
+    :tanagra_auth_disable_checks => "true",
+    :tanagra_auth_bearer_token => "false",
+    :tanagra_underlay_files => "aou/test/SC2023Q3R1/SC2023Q3R1.json,aou/test/SR2023Q3R1/SR2023Q3R1.json",
```
Comment (chenchals) [inline api/libproject/environments.rb:53]: nice!

**Review by chenchals — COMMENTED** [review]
clarifications

File: tanagra-aou-utils/tanagra_env_variables_template.yaml (hunk 1 above)
Comment (freemabd) [reply tanagra-aou-utils/tanagra_env_variables_template.yaml:14]: API starts with a null pointer exception if this is left out. I'm going to bring it up with Mariko

**Pull request merged**